kairon/shared/chat/processor.py

```
@@ -36,6 +36,10 @@ def save_channel_config(configuration: Dict, bot: Text, user: Text):
             from kairon.shared.channels.mail.processor import MailProcessor
             if MailProcessor.check_email_config_exists(bot, configuration['config']):
                 raise AppException("Email configuration already exists for same email address and subject")
+            input_interval = int(configuration["config"]["interval"])
+            system_interval = int(Utility.environment['integrations']["email"]['interval'])
+            if input_interval < system_interval:
+                raise AppException(f"Minimum Interval should be greater than equal to {system_interval} minutes")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)